src/test/java/net/snowflake/ingest/streaming/internal/OAuthBasicTest.java

```diff
@@ -114,7 +118,7 @@ public void testCreateOAuthClient() throws Exception {
   @Test
   public void testSetRefreshToken() throws Exception {
     SnowflakeStreamingIngestClientInternal<StubChunkData> client =
-        new SnowflakeStreamingIngestClientInternal<>("TEST_CLIENT");
+        new SnowflakeStreamingIngestClientInternal<>("TEST_CLIENT", false);
```

Comment:
please add a code comment here with `SNOW-1490151` in the comment so that whenever that JIRA is fixed we have an easy way of finding all the places that need fixes.

Reply:
Added